pymfe/mfe.py

```diff
@@ -1287,7 +1327,7 @@ def metafeature_description(
         deps = _internal.check_group_dependencies(groups)
 
```

Comment:
Maybe add a Exception when the user enables both sort-related arguments, to avoid confusion

Reply:
I think we can use both together if you want to sort both columns.

Reply:
I added warnings and tests in 1fb8844